node/libs/roles/src/attester/tests.rs

```diff
@@ -141,23 +141,37 @@ fn test_batch_qc() {
     let setup1 = Setup::new(rng, 6);
     let setup2 = Setup::new(rng, 6);
     let mut genesis3 = (*setup1.genesis).clone();
-    genesis3.attesters = Committee::new(setup1.genesis.attesters.iter().take(3).cloned()).unwrap();
+    genesis3.attesters = Committee::new(
+        setup1
+            .genesis
+            .attesters
+            .as_ref()
+            .unwrap()
+            .iter()
+            .take(3)
```

Comment:
setup1.genesis.attesters.as_ref().unwrap()[3].cloned()

Reply:
I believe it's not quite the same. The current implementation selects the first three attesters from the genesis, whereas with the new line, we would only pick the one in the third position. Although I don't see a problem in just picking one, since the genesis is only used for an error case for the verification. Perhaps that was the original idea behind the comment. I'll change it, and we can revert if necessary.

Reply:
oh, you are right. I misread the code. It was ok in the original form.